### PR TITLE
PEP8 fixes

### LINKED ISSUE #20: PEP8
Comply with https://www.python.org/dev/peps/pep-0008/

### PR DESCRIPTION
**Pull request by delirious-lettuce**
This PR fixes #20

* Group imports according to PEP8
* Remove unused imports
* Rename functions/variables using `snake_case`
* Fix typo: `insted` -> `instead`
* Two blank lines above top-level definitions
* Reformat lines > 79 characters long

### CODE REVIEW

File: timeutils.py [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
 HS_TZ = timezone(timedelta(hours=2))
 
+
 def meetup_time(dt):
     """Converts time to format accepted by meetup API"""
     logger.debug("%s, %s", dt, dt.timestamp())
     logger.info('change timezone')
     dt = dt.replace(tzinfo=HS_TZ)
     logger.debug("%s, %s", dt, dt.timestamp())
-    result = datetime2ms(dt)
```
Comment (not7cd) [inline timeutils.py]: I don't think it's uncommon to see such names

File: core.py [hunk 2]
```diff
@@ -1,30 +1,29 @@
 import logging
+
 import meetup.api
-from datetime import datetime, timedelta
+
 from config import target
 from timeutils import meetup_time, meetup_duration
```
Comment (not7cd) [inline core.py]: Can't find anything in PEP8 about blank lines separating imports

Comment (delirious-lettuce) [reply core.py]: Imports should be grouped in the following order:

1. standard library imports
2. related third party imports
3. local application/library specific imports

You should put a blank line between each group of imports.

* http://pep8.org/#imports
---

If you use PyCharm, you can just go to `Code > Optimize Imports` to get the same results.

File: timeutils.py (hunk 1 above)
Comment (delirious-lettuce) [reply timeutils.py]: * http://pep8.org/#descriptive-naming-styles

**Pull request merged**